test/SparqlAntlrParserTest.cpp

```diff
@@ -266,12 +270,12 @@ TEST(SparqlParser, ComplexConstructTemplate) {
       parse<&Parser::constructTemplate>(input),
       m::ConstructClause(
           {{Blank("0"), Var("?a"), Blank("3")},
-           {Blank("1"), Iri(first), Blank("2")},
-           {Blank("1"), Iri(rest), Iri(nil)},
-           {Blank("2"), Iri(first), Var("?c")},
+           {Blank("2"), Iri(first), Blank("1")},
            {Blank("2"), Iri(rest), Iri(nil)},
+           {Blank("1"), Iri(first), Var("?c")},
+           {Blank("1"), Iri(rest), Iri(nil)},
            {Blank("3"), Iri(first), Var("?b")},
-           {Blank("3"), Iri(rest), Blank("1")},
+           {Blank("3"), Iri(rest), Blank("2")},
```

Comment:
Tags 1 and 2 are swapped here, because changing the "iterative approach" in the qlever visitor to `visitVector` changes the order of tree traversal